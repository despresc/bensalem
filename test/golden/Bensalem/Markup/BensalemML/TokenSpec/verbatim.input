\```|Really, anything can appear in here, like \element[]{} and \%%comment
  and \\\{\}\[\] and \#lvl and even lexically invalid things like \ escaped space
|```/
and we go back to text afterward \element[]{}
