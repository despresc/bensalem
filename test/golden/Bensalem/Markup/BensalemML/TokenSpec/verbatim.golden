(0,1,1)(5,1,6) StartVariableVerb 5
(5,1,6)(12,1,13) VariableVerbPlainText "Really,"
(12,1,13)(13,1,14) LineSpace 1
(13,1,14)(21,1,22) VariableVerbPlainText "anything"
(21,1,22)(22,1,23) LineSpace 1
(22,1,23)(25,1,26) VariableVerbPlainText "can"
(25,1,26)(26,1,27) LineSpace 1
(26,1,27)(32,1,33) VariableVerbPlainText "appear"
(32,1,33)(33,1,34) LineSpace 1
(33,1,34)(35,1,36) VariableVerbPlainText "in"
(35,1,36)(36,1,37) LineSpace 1
(36,1,37)(41,1,42) VariableVerbPlainText "here,"
(41,1,42)(42,1,43) LineSpace 1
(42,1,43)(46,1,47) VariableVerbPlainText "like"
(46,1,47)(47,1,48) LineSpace 1
(47,1,48)(59,1,60) VariableVerbPlainText "\\element[]{}"
(59,1,60)(60,1,61) LineSpace 1
(60,1,61)(63,1,64) VariableVerbPlainText "and"
(63,1,64)(64,1,65) LineSpace 1
(64,1,65)(74,1,75) VariableVerbPlainText "\\%%comment"
(74,1,75)(77,2,3) Indent "\n  "
(77,2,3)(80,2,6) VariableVerbPlainText "and"
(80,2,6)(81,2,7) LineSpace 1
(81,2,7)(91,2,17) VariableVerbPlainText "\\\\\\{\\}\\[\\]"
(91,2,17)(92,2,18) LineSpace 1
(92,2,18)(95,2,21) VariableVerbPlainText "and"
(95,2,21)(96,2,22) LineSpace 1
(96,2,22)(101,2,27) VariableVerbPlainText "\\#lvl"
(101,2,27)(102,2,28) LineSpace 1
(102,2,28)(105,2,31) VariableVerbPlainText "and"
(105,2,31)(106,2,32) LineSpace 1
(106,2,32)(110,2,36) VariableVerbPlainText "even"
(110,2,36)(111,2,37) LineSpace 1
(111,2,37)(120,2,46) VariableVerbPlainText "lexically"
(120,2,46)(121,2,47) LineSpace 1
(121,2,47)(128,2,54) VariableVerbPlainText "invalid"
(128,2,54)(129,2,55) LineSpace 1
(129,2,55)(135,2,61) VariableVerbPlainText "things"
(135,2,61)(136,2,62) LineSpace 1
(136,2,62)(140,2,66) VariableVerbPlainText "like"
(140,2,66)(141,2,67) LineSpace 1
(141,2,67)(142,2,68) VariableVerbPlainText "\\"
(142,2,68)(143,2,69) LineSpace 1
(143,2,69)(150,2,76) VariableVerbPlainText "escaped"
(150,2,76)(151,2,77) LineSpace 1
(151,2,77)(156,2,82) VariableVerbPlainText "space"
(156,2,82)(157,3,1) Indent "\n"
(157,3,1)(162,3,6) EndVariableVerb 5
(162,3,6)(163,4,1) Indent "\n"
(163,4,1)(166,4,4) PlainText "and"
(166,4,4)(167,4,5) LineSpace 1
(167,4,5)(169,4,7) PlainText "we"
(169,4,7)(170,4,8) LineSpace 1
(170,4,8)(172,4,10) PlainText "go"
(172,4,10)(173,4,11) LineSpace 1
(173,4,11)(177,4,15) PlainText "back"
(177,4,15)(178,4,16) LineSpace 1
(178,4,16)(180,4,18) PlainText "to"
(180,4,18)(181,4,19) LineSpace 1
(181,4,19)(185,4,23) PlainText "text"
(185,4,23)(186,4,24) LineSpace 1
(186,4,24)(195,4,33) PlainText "afterward"
(195,4,33)(196,4,34) LineSpace 1
(196,4,34)(204,4,42) InlineTag "element"
(204,4,42)(205,4,43) StartAttrSet
(205,4,43)(206,4,44) EndAttrSet
(206,4,44)(207,4,45) StartBraceGroup
(207,4,45)(208,4,46) EndBraceGroup
(208,4,46)(209,5,1) Indent "\n"
